spaceship.zsh

```diff
@@ -151,7 +151,9 @@ spaceship_prompt() {
   RETVAL=$?
 
   # Should it add a new line before the prompt?
-  [[ $SPACESHIP_PROMPT_ADD_NEWLINE == true ]] && echo -n "$NEWLINE"
+  [[ $UID == 0 ]] && SPACESHIP_PROMPT_NEED_NEWLINE=true
```

Comment:
Please, add a reference to the issue in a comment above. This looks like a hack, so new contributors should know why we have it.